zerver/tests/test_user_status.py

```diff
@@ -185,6 +195,48 @@ def update_status_and_assert_event(
         self.assert_json_success(result)
         self.assertEqual(events[0]["event"], expected_event)
 
+    def test_clear_statuses_upon_expiry(self) -> None:
+        client1 = get_client("web")
+        hamlet = self.example_user("hamlet")
+        scheduled_clear_status_datetime = datetime_to_timestamp(timezone_now())
+        scheduled_end_time_expired = scheduled_clear_status_datetime - 5 * 60
+
+        # Set a user status with some end time.
+        update_user_status(
+            user_profile_id=hamlet.id,
+            status_text="out to lunch",
+            emoji_name="car",
+            emoji_code="1f697",
+            reaction_type=UserStatus.UNICODE_EMOJI,
+            client_id=client1.id,
+            scheduled_end_time=scheduled_end_time_expired,
+        )
+        rec_count = UserStatus.objects.filter(user_profile_id=hamlet.id).count()
```

Comment:
It's not clear what `rec` stands for.

Reply:
It indicates a row is successfully created in the UserStatus table, when `scheduled_end_time` is set. It may not have relevance since I anyways have statement verifying that status is set in the next line. Do you want this out?

Reply:
I meant to just name the `rec_count` variable correctly. `row_count` or anything.

We generally avoid short forms like `msg` for message in code.

Reply:
sure! there are other `rec_count` variables in the file. Do you want a prep commit to maintain consistency?

Reply:
Oh, okay!

I'll leave it for Tim to answer. Maybe it's only me who don't think `rec` is good. I'd have used the term record or row. Personal preference. -- Not an issue to be much worried about.